### DISCUSSION TITLE
please help me

### DISCUSSION THREAD

**Discussion by khoatran123-123** [community/community#79306] [GitHub Education]
### Hi everyone,

I have a problem to enroll education program. This issue is **Your application cannot be reviewed until you fix the following:
Your uploaded proof has already been used previously.**

**Comment by manik-007** [comment]
Hello @khoatran123-123 

I have the exact solution for this problem. Few days back this same thing happened with my friend.
Just Click the picture again so that its details changed like : name, date created etc. Then upload it. I am 100% sure the same document will get accepted.

If this helped you in any way then do mark my solution as your answer. It'll be a great favour :)
And if faced any problem then don't hesitate to ask. I am here to help you out.

**Comment by timred01** [comment]
maybe you have problem on your account github
